addons/website_sale/controllers/main.py

```diff
@@ -1611,6 +1611,7 @@ def update_cart_address(self, partner_id, mode='billing', **kw):
         ):
             raise Forbidden()
 
+        order_sudo.access_point_address = {}
```

Comment:
Could you please move it to L1628 as it makes sense to reset pickup location only if delivery address is changed not a billing one?